Sources/Components/Checkbox/Checkbox.swift

```diff
@@ -45,7 +45,21 @@ extension Warp {
             self.extraContent = extraContent
             self.action = action
         }
-        
+
+        /// Initializes a new `Checkbox` without title and extra content.
+        ///
+        /// - Parameters:
+        ///   - isSelected: A `Binding` value indicating whether the checkbox button is selected.
+        ///   - style: The style the checkbox can have (default, error, disabled).
+        public init(isSelected: Binding<Bool>,
```

Comment:
Can we achieve the same thing by just adding a default value on line 41?
`action: @escaping () -> Void = {}) {`

Reply:
Of course we can, but I found this solution a bit more descriptive. Do you prefer making the `title` optional with default `nil` and the action with a default empty closure? 🙂

Reply:
I would go with one initializer, easier to document with the structure we have in our doc.

Reply:
OK, I've made the changes according to that.

Reply:
Looks great thanks 👍
Feel free to update docs also if you know how to do it.
https://github.com/warp-ds/docs/blob/main/docs/components/checkbox/ios.md
https://github.com/warp-ds/docs/blob/1b36bb81ba3e23ec6c0709e480cc01d086931380/docs/api-table.js#L3307
Otherwise I can do it.

How about `Radio` component? Should we do the same?

Reply:
Uhh, I don't really know that documentation format. 😬 Could you do the changes?

Regarding the radio it's not urgent at this time, but I'll probably do it soon.

Reply:
Amazing, yes I will update the docs

Reply:
are you releasing this change also?

Reply:
Do we have a script or automation for that? 🙂

Reply:
Nope, fully fully manual :D